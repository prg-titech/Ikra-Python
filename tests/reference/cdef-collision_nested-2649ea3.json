{
    "basename": "cdef-collision_nested-2649ea3",
    "cmd": "new-src/interface.py --emit-cdef {infile}",
    "infile": "examples/collision/collision_nested.py",
    "infile_hash": "32b6014a5cbf9597cb4b52c12e91d20c646ad1a3138607b7120da6e7",
    "stdout": "cdef-collision_nested-2649ea3.stdout",
    "stdout_hash": "8cbf5102868c18d596d4220480db29298b46c3e741b531586dfc9fe5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}