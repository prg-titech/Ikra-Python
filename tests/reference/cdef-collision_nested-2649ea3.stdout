
int Body_Body_compute_force();
int Body_Body_body_update();
int Body_Body_initialize_merge();
int Body_Body_prepare_merge();
int Body_Body_update_merge();
int Body_Body_delete_merged();
int parallel_new_Body(int object_num);
int AllocatorInitialize();
int Body_do_all(void (*pf)(int, int, float, float, float, float, float, float, float, bool, bool));
int AllocatorUninitialize();
